src/main/java/jenkins/plugins/nodejs/tools/NodeJSInstallation.java

```diff
@@ -78,6 +88,16 @@ private File getExeFile() {
         return new File(nodeJSHome, "bin/" + execName);
     }
 
+    String getBinFolder() {
+        final Computer computer = Computer.currentComputer();
+
+        // TODO post 1.624 use Computer#isUnix
```

Comment:
You can just require a `unix` method parameter, and use `Launcher.isUnix` from the caller.